webui/react/src/components/ResourcePoolCard.tsx

```diff
@@ -43,88 +44,76 @@ export const rpLogo = (type: V1ResourcePoolType): React.ReactNode => {
   return <img src={iconSrc} />;
 };
 
-const rpAttrs = [
-  [ 'location', 'Location' ],
-  [ 'instanceType', 'Instance Type' ],
-  [ 'preemptible', 'Spot/Preemptible' ], // FIXME do we want this spot or details.spot_enabled
-  [ 'minAgents', 'Min Agents' ],
-  [ 'maxAgents', 'Max Agents' ],
-  [ 'slotsPerAgent', 'Slots Per Agent' ],
-  [ 'auxContainerCapacityPerAgent', 'Max Aux Containers Per Agent' ],
-  [ 'schedulerType', 'Scheduler Type' ],
+const poolAttributes = [
+  { key: 'location', label: 'Location' },
+  { key: 'instanceType', label: 'Instance Type' },
+  // FIXME for preemptible, do we want this spot or details.spot_enabled?
```

Comment:
unrelated: I think we can remove this comment. we just want to indicate that this preemption here is talking about instance preemption and not job/task preemption.